5-Heroku/server/models/todoModel.js

```diff
@@ -0,0 +1,85 @@
+const mongoose = require('mongoose')
+const todoScheme = new mongoose.Schema({
```

Comment:
general note regarding this file. this is what we call a DAO (data access object) it's usually a nice interface to the DB (exactly what u did)
so I would call it todoDAO instead of todoModel.

Reply:
just name convention. DAO is the facade/interface to the db (later u can replace your db or queries and it keeps the rest of the application clean)

we also have DTO (data transfer object) this is what we call to the json that is passed between the server and the client.